meow/frontend/src/components/EditPost/Content.js

```diff
@@ -15,7 +15,7 @@ class EditPost extends React.Component {
   };
 
   handleOk = () => {
-    this.props.save();
+    this.props.savePost();
```

Comment:
Good! We'll have to change my commit back to passing the function in as `savePost` rather than `save`, which is the better way to go anyways :+1: